core/templates/dev/head/admin/admin.html

```diff
@@ -11,93 +11,203 @@
+        <div class="navbar-header protractor-test-navbar-header pull-left">
+          <a class="oppia-navbar-brand-name oppia-transition-200" href="/gallery">
+            <img src="/images/logo/288x128_logo_white.png" class="oppia-logo" ng-class="'oppia-logo-wide'">
+          </a>
+          <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+            <li>
+              <span class="oppia-navbar-breadcrumb-separator"></span>
+              Admin
+            </li>
+          </ul>
+        </div>
 
-    <h3>Reload a single exploration</h3>
-    <div class="container" style="margin-left: 0;">
-      {% for exploration in demo_explorations %}
-        <div class="row protractor-test-reload-exploration-row">
-          <span class="col-lg-4 col-md-4 col-sm-4 protractor-test-reload-exploration-title">
-            {{exploration[1]}}
-          </span>
-          <span class="col-lg-2 col-md-2 col-sm-2">
-            <button type="button" class="btn btn-default protractor-test-reload-exploration-button" ng-click="reloadExploration({{exploration[0]}})">
-              Reload
-            </button>
-          </span>
+        {% if nav_mode != 'signup' %}
+        <div ng-cloak class="navbar-header pull-right">
+          <ul class="nav oppia-navbar-nav oppia-navbar-profile-admin">
+            {% if username %}
+              <li class="dropdown pull-right">
+                <a class="dropdown-toggle oppia-navbar-dropdown-toggle" data-toggle="dropdown" ng-mouseover="onMouseoverProfilePictureOrDropdown($event)" ng-mouseleave="onMouseoutProfilePictureOrDropdown($event)">
+                  <div class="oppia-navbar-profile-picture-container" ng-cloak>
+                    {% if profile_picture_data_url %}
+                      <img src="{{profile_picture_data_url}}" class="oppia-navbar-profile-picture img-circle">
+                      <span class="caret" style="margin-top: 10px;"></span>
+                    {% else %}
+                      <i class="material-icons md-40" style="margin-top: -1px;">&#xE853;</i>
+                      <span class="caret" style="margin-top: -26px;"></span>
+                    {% endif %}
+
+                    <div class="oppia-navbar-dashboard-indicator ng-cloak" ng-if="numUnseenNotifications > 0">
+                      <span class="oppia-navbar-dashboard-indicator-text">
+                        <[numUnseenNotifications]>
+                      </span>
+                    </div>
+
+                    <div style="display: none;" class="oppia-user-email">
+                      {{user_email}}
+                    </div>
+
+                    {% if is_admin or is_moderator %}
+                      <div class="oppia-navbar-role-indicator">
+                        {% if is_admin %}
+                          <!-- "right: 4px;" is necessary here but not in moderator to prevent 'A' from appearing off-center because 'A' is slightly thinner than 'M' in this font -->
```

Comment:
Isn't this a global change that should be made generally on the site? If so, let's not fix it here, and instead, let's file a separate issue for it.